doc/manual/decoding.rst

```diff
@@ -17,9 +17,11 @@ Basic Estimators
 
 Scaler
 ^^^^^^
-This will standardize data across channels. Each channel type (mag, grad or eeg) is treated separately. During training time, the mean (`ch_mean_`) and standard deviation (`std_`) is computed in the ``fit`` method and stored as an attribute to the object. The ``transform`` method is called to transform the training set. To perform both the ``fit`` and ``transform`` operations in a single call, the ``fit_transform`` method may be used. During test time, the stored mean and standard deviation are used in the ``transform`` method. To recover the original data, you can use ``inverse_transform``.
+This will scale data across channels. Each channel type (mag, grad or eeg) is treated separately. Users can specify the method to scale with. During training time, ``scikit-learn`` objects are created if scaling method is ``StandardScaler`` or ``RobustScaler``. The ``transform`` method is called to transform the training set. To perform both the ``fit`` and ``transform`` operations in a single call, the ``fit_transform`` method may be used. During test time, the stored scaling values are used in the ``transform`` method. To recover the original data, you can use ``inverse_transform``.
 
-.. note:: This is different from the ``StandarScaler`` estimator offered by Scikit-Learn. The ``StandardScaler`` standardizes each feature, whereas the ``Scaler`` object standardizes by channel type.
+.. note:: For working of every scaling method, atleast scikit-learn version 0.18 is required.
+
+.. note:: This is different from directly applying ``StandarScaler`` or ``RobustScaler``  estimator offered by Scikit-Learn. The ``StandardScaler`` or ``RobustScaler`` scales each feature, whereas the ``Scaler`` object scales by channel type.
```

Comment:
I was wrong, they are still different (see other reply). I'm also about to push an updated `manual` entry that hopefully clarifies it.